ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -124,385 +129,392 @@ Control {
+                        Global.openLinkWithConfirmation(link, StatusQUtils.StringUtils.extractDomainFromLink(link));
+                    }
+
+                    width: GridView.view.cellWidth - Style.current.padding
+                    height: GridView.view.cellHeight - Style.current.padding
+
+                    HoverHandler {
+                        id: hoverHandler
+                        cursorShape: hovered ? Qt.PointingHandCursor : undefined
+                    }
+                    StatusRoundedImage {
+                        id: collectibleImage
+                        anchors.fill: parent
                         color: !!model.backgroundColor ? model.backgroundColor : "transparent"
                         radius: Style.current.radius
                         showLoadingIndicator: true
                         isLoading: image.isLoading || !model.imageUrl
                         image.fillMode: Image.PreserveAspectCrop
                         image.source: model.imageUrl ?? ""
-
-                        Control {
-                            width: (amountText.contentWidth + Style.current.padding)
-                            height: 24
-                            anchors.left: parent.left
-                            anchors.leftMargin: 12
-                            anchors.top: parent.top
-                            anchors.topMargin: 12
-                            //TODO TBD, we need to show the number if the user has more than 1 of each collectible
-                            //not sure how to name the role
-                            visible: (model.userHas > 1)
-
-                            background: Rectangle {
-                                radius: 30
-                                color: Theme.palette.indirectColor2
+                        TapHandler {
+                            acceptedButtons: Qt.LeftButton | Qt.RightButton
+                            onSingleTapped: {
+                                if ((eventPoint.event.button === Qt.LeftButton) && (model.communityId !== "")) {
+                                    Global.openPopup(visitComunityPopupComponent, {communityId: model.communityId, communityName: model.communityName,
+                                                                                   communityLogo: model.communityImage, tokenName: model.name,
+                                                                                   tokenImage: model.imageUrl, isAssetType: false });
+                                } else {
+                                    if (eventPoint.event.button === Qt.LeftButton) {
+                                        delegateItem.openCollectibleURL()
+                                    } else {
+                                        if (model.communityId !== "") {
+                                            Global.openMenu(delegatesActionsMenu, collectibleImage, { communityId: model.communityId, context: "collectibles"});
```

Comment:
I've seen these strings "collectibles" and other ones used for this `context` property around this file in so many places.. Maybe we could define a readonly property for them, it will be safer and less bug prompt.